data/pop-upgrade.sh

```diff
@@ -1,8 +1,13 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
There are loads of unquoted variables here. I think this can be taken care of post-merge